state.go

```diff
@@ -282,15 +283,16 @@ func NewWaitState(id MenderState, t Transition) WaitState {
 	return &waitState{
 		baseState: baseState{id: id, t: t},
 		cancel:    make(chan bool),
-		wakeup:    make(chan bool),
+		// wakeup: is a global channel shared between all states, defined in TODO
```

Comment:
Will resolve if you agree that stateContext is the best place to keep the global wakeup channel.

Reply:
Yep, that's even better than my original suggestion to make it truly global.